les/serverpool.go

```diff
@@ -657,6 +656,28 @@ func (pool *serverPool) checkDialTimeout(entry *poolEntry) {
 	pool.setRetryDial(entry)
 }
 
+func (pool *serverPool) Info() []*poolEntryInfo {
+	stateStringMap := make(map[int]string)
```

Comment:
Could this be extracted as a mapping constructed as a map literal at the package scope instead of doing it inline here?